app/code/Magento/Backend/Model/Validator/IpValidator.php

```diff
@@ -5,6 +5,8 @@
  */
 namespace Magento\Backend\Model\Validator;
 
+use IPTools\Network;
```

Comment:
Seems like dependency should be added to module's composer.json file

Reply:
Sorry, I saw the files were moved in the most recent merge, but forgot to add the dependency here. I'll get that sorted.